shared/constants/snaps/permissions.ts

```diff
@@ -8,6 +8,7 @@ export const EndowmentPermissions = Object.freeze({
   'endowment:lifecycle-hooks': 'endowment:lifecycle-hooks',
   ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
   'endowment:page-home': 'endowment:page-home',
+  'endowment:signature-insight': 'endowment:signature-insight',
```

Comment:
If we are activating this permission it needs to specify a permission name, icon etc

Reply:
Yeah, going to work on this part with @eriknson

Reply:
Done.